staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -354,7 +360,9 @@ type VirtualMachineInstanceMigrationState struct {
 	TargetNodeDomainDetected bool `json:"targetNodeDomainDetected,omitempty"`
 	// The address of the target node to use for the migration
 	TargetNodeAddress string `json:"targetNodeAddress,omitempty"`
+	// +kubebuilder:validation:Type=object
 	// The list of ports opened for live migration on the destination node
+	// +k8s:openapi-gen=false
```

Comment:
is there a reason openapi-gen=false here?

Reply:
And that's a problem to discuss (totally forgot about this). map[int]int is invalid type. It would be bad to not generate this. We can work around this by declaring OpenAPISchemaType() or we can change our code to map[string]int.

Reply:
OH, that's super interesting. I just hit the same thing when I was messing with the generation logic myself. 

How exactly would the workaround look with OpenAPISchemaType ?

Reply:
Actually I tried it and it didn't work... so https://github.com/kubevirt/kubevirt/pull/3375#issuecomment-624057689 if I didn't miss anything...

Reply:
can we update this now that the ports map has been fixed? once this is removed as well as the kubebuilder markers, i think we'll be really close.

Reply:
Yes, already working on this. Also, I rebased and found more `+k8s:openapi-gen=false ` which should be there.